packages/react-components/src/components/CollapsiblePanel/CollapsiblePanel.jsx

```diff
@@ -1,102 +0,0 @@
-import PropTypes from 'prop-types';
-import React from 'react';
-import Scroll from 'react-scroll';
```

Comment:
With this happening I think we can remove `react-scroll` and close https://github.com/department-of-veterans-affairs/va.gov-team/issues/34437.